platform-api/tests/test_platform.py

```
@@ -0,0 +1,108 @@
+            
+        async def post(self, url, **kwargs):
+            if "deploy_hooks" in url:
+                return MockResponse({}, 200)
+            if "graphql" in url:
+                return MockResponse({"data": {"viewer": {"accounts": [{"workersInvocationsAdaptive": [{"sum": {"requests": 10}, "dimensions": {"datetime": "2026-01-01"}}]}]}}})
+            return MockResponse({})
+
+    # Patch the HTTP client across the router and service
+    import services.cf_client
+    import routers.platform
+    monkeypatch.setattr(services.cf_client, "get_client", lambda **kw: MockAsyncClient())
```

Ruff (0.15.11)

[warning] Unused lambda argument: `kw`

(ARG005)